wrapper/src/test/java/integration2/DriverHelper.java

```diff
@@ -0,0 +1,348 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package integration2;
+
+import com.mysql.cj.conf.PropertyKey;
+import java.sql.Driver;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.postgresql.PGProperty;
+import org.testcontainers.shaded.org.apache.commons.lang3.NotImplementedException;
+import integration2.container.TestDriver;
+import integration2.container.TestEnvironment;
+
+public class DriverHelper {
+
+  private final static Logger LOGGER = Logger.getLogger(DriverHelper.class.getName());
+
+  public static String getDriverProtocol() {
+    return getDriverProtocol(
+        TestEnvironment.getCurrent().getInfo().getRequest().getDatabaseEngine(),
+        TestEnvironment.getCurrent().getCurrentDriver());
+  }
+
+  public static String getDriverProtocol(DatabaseEngine databaseEngine) {
```

Comment:
NIT: getDriverProtocolURL ?

Reply:
No, it's just a driver protocol, not a complete url.